packages/contracts-bedrock/test/kontrol/scripts/run-kontrol.sh

```diff
@@ -36,9 +36,10 @@ kontrol_prove() {
     $break_on_calls \
     $break_every_step \
     $tests \
-    --init-node-from $state_diff \
+    --init-node-from-diff $state_diff \
     --kore-rpc-command 'kore-rpc-booster --equation-max-recursion 100' \
-    --xml-test-report
+    --xml-test-report \
+    --bmc-depth 1
```

Comment:
> Introduces `--bmc-depth 1` as part of the `kontrol prove` options as a workaround until the `copy_memory_to_memory` lemma is updated

What's the summary on why this resolves the failures, and are there risks/tradeoffs here?

Reply:
It's a way of not fully executing the loops (in this case, the one in `copy_memory_to_memory`). Instead of fully summarizing the loop's behavior, we only execute one iteration. The tradeoff is that it's a less rigorous reasoning technique than having the summary in place.
Raoul's [excellent recent post](https://runtimeverification.com/blog/formally-verifying-loops-part-1) explains this in fantastic detail!

Reply:
Resolving this since the `copy_memory_to_memory` summary has been updated!